code/API_definitions/device-status.yaml

```diff
@@ -17,11 +83,11 @@ security:
   - oAuth2ClientCredentials: []
   #  - BasicAuth: []
   #  - apiKey: []
-  - three_legged:
-      - device-status-roaming-read
-      - device-status-roaming-event
+  - threeLegged:
+      - device-status:roaming:read
+      - device-status:roaming-event-subscriptions:create
```

Comment:
How are we going to manage the scopes for the subscriptions? I can see that we have an scope for creating subscriptions but I think we are missing an additional scope to delete subscriptions too. And, in order to retrieve the available subscriptions, are we going to use the same scope as we use to read the roaming status?

```suggestion
      - device-status:roaming:read
      - device-status:roaming-event-subscriptions:read
      - device-status:roaming-event-subscriptions:create
      - device-status:roaming-event-subscriptions:delete
```

Whether we decide to use specific scopes or not, we should use the `security` property for each individual operation because right now, using the global property some operations don't have a correct scope. For example we have a create scope for a delete operation:

![image](https://github.com/camaraproject/DeviceStatus/assets/33178221/062c91eb-e6ca-4b13-95a9-cd698c82fe98)